src/views/tutorial-library/components/filter-section/index.tsx

```diff
@@ -0,0 +1,19 @@
+import { PropsWithChildren } from 'react'
+
+import filterSectionStyle from './filter-section.module.css'
+
+export type FilterSectionProps = PropsWithChildren<{
+	/**
+	 * Heading to be rendered above the section
+	 */
+	heading: string
+}>
+
+export function FilterSection({ heading, children }: FilterSectionProps) {
+	return (
+		<section className={filterSectionStyle.root}>
+			<span className={filterSectionStyle.heading}>{heading}</span>
+			<ul className={filterSectionStyle.filterList}>{children}</ul>
+		</section>
```

Comment:
I'm not sure if it makes sense to use a `section` element here, but it seems okay per https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section#using_a_section_without_a_heading

Reply:
Also I should probably use the `Text` component here 💭